foundry.toml

```diff
@@ -14,7 +14,7 @@ libs = ['lib', 'node_modules/@openzeppelin', 'node_modules/hardhat']
 solc = '0.8.25'
 use_literal_content = true
 extra_output = ["devdoc", "userdoc", "storagelayout"]
-evm_version = 'london'
+evm_version = 'istanbul'
```

Comment:
Simulation shows that current gas stipend config on ronin-mainnet's `RoninValidatorSet` ([Reference here](https://app.roninchain.com/address/0x617c5d73662282ea7ffd231e020eca6d2b0d552f?t=contract&st=read-contract-proxy) might throws `Out of Gas` error if `delegated` by migrator contract with higher evm version (i.e london).